go-app-ussd.js

```diff
@@ -66,12 +66,17 @@ go.app = function() {
         });
 
         self.states.add('states:select_language', function(name){
+            var language_previously_not_set = self.im.user.lang === null;
```

Comment:
@hodgestar not sure if this is good practice, but it was the only way I could check if the language had previously been set.